src/main/java/com/company/model/Employee.java

```diff
@@ -0,0 +1,62 @@
+package com.company.model;
+
+import java.util.function.Function;
+
+public class Employee {
+    private final int id;
+    private static int count = 1;
+    private final String firstname;
```

Comment:
Check variables' naming according to conventions. If a variable consists of more than one word, capitalize the first letter of each subsequent word. (https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html)